js/components/uswds_radio_widget.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+const USWDSRadioWidget = function(props) {
+  console.log('props for radio: ', props);
+  const { enumOptions, inline } = props.options;
+
+return (
+    <fieldset className="usa-fieldset-inputs field-radio-group">
+      <ul class="usa-unstyled-list">
+      {enumOptions.map((option, i) => {
+        console.log('option: ', option);
+        const radio = (
+            <input
+              type="radio"
+              id={`${props.label}-${option.label}`}
```

Comment:
Isn't it possible this will have spaces in it? Does that matter?

Reply:
I thought `label` was dash-separated and `title` was space-separated, but I'll double-check.

Reply:
Okay, so the deal with this is that currently the radio widget only accepts yes/no as options (these show up as `option.label`). It needs some tweaking to accept custom labels, which I don't want to mess with until we're sure we need them. 

The `props.label` piece is really the `name` from the markdown, which is currently underscore-separated. So this comes out looking something like: `request_fee_waiver-no`.

I think this is okay for now, but if we change the `option.label`, we'll need to revisit.